### DISCUSSION TITLE
Require MFA by default

### DISCUSSION THREAD

**Discussion by ghost** [dani-garcia/vaultwarden#3455] [Ideas]
Currently we can set organization policy to require members have MFA active, but we also want to require users who are not a member of any organization to require MFA by default at a more global level.

Is there a way to accomplish this?

**Comment by BlackDex** [comment]
There is no account creation flow to force this.
And setting MFA needs access to the web-vault.

So basically, no there is no way to accomplish this.

**Reply by ghost** [reply]
What a shame... Thanks for the explanation!
